### ISSUE TITLE
Allow blacklists in config.hcl

### ISSUE THREAD

**Issue by TinLe** [cloudquery/cloudquery#889]
Currently config.hcl allow one to give specific project or folder, along with all ('*').  Same for list of resources to fetch, e.g. you can list a bunch of resources.

It would be nice to be able to do the reverse, e.g. blacklist.

For project, we can give a list of one or more projects and folder to not fetch data from.

For resources, we can give a list of one or more resources to not fetch data from.

### Is your feature request related to a problem? Please describe.
When you have many projects and resources, it can be painful to list all projects EXCEPT for the blacklisted one that you do not want.  The same applies to resources, when you have to list all resources except for the one or two that you do not want.

### Describe the solution you'd like
An example using GCP provider config:

provider "gcp" {
  configuration {
   // Nice to be able to put blacklisted folders here, so fetch data from all folders except these.
    // folder_ids_blacklist = [ "organizations/<ORG_ID>", "folders/<FOLDER_ID>" ]

    // project_ids_blacklist = [<CHANGE_THIS_TO_YOUR_PROJECT_ID>]
  }

  // list of resources to NOT fetch
  resources_blacklist = [
    "resource_manager.projects",
  ]
}

### Describe alternatives you've considered
Without blacklist, the current solution is to list all projects, except for the blacklisted ones that we do not want to fetch.  This does not work for us as we have thousands of projects.

Same idea for resources.  Very painful.

### Additional context
It's a nice to have, make configuration easier to manage.

**Comment by bbernays** [comment]
@TinLe we do already support the `not_resources` Attribute for specifying which resources not to fetch. I am working on the documentation for it tomorrow, so once that is done I will send you a link to it

**Comment by TinLe** [comment]
@bbernays Did you meant "skip_resources" ?  I found that in the code.  I did not realize that option exists.  Thank you for pointing me to it.

It would be nice also to have something similar for projects and folders.  We have too many projects, and it is painful to list all the ones we want to scan vs just listing the smaller number that we do not want.

**Comment by bbernays** [comment]
Seeing as we already support what you are looking for at the core level (skipping of resources) I am going to move this feature request to the GCP provider so that we can look into supporting skipping projects/folders
